conf/openmetadata.yaml

```diff
@@ -151,6 +151,26 @@ authenticationConfiguration:
   callbackUrl: ${AUTHENTICATION_CALLBACK_URL:-""}
   jwtPrincipalClaims: ${AUTHENTICATION_JWT_PRINCIPAL_CLAIMS:-[email,preferred_username,sub]}
   enableSelfSignup : ${AUTHENTICATION_ENABLE_SELF_SIGNUP:-true}
+  samlConfiguration:
```

Comment:
shouldn't this part of authentication configuration. Instead of having another config section

Reply:
This is under authentication configuration!